src/index.rs

```diff
@@ -87,12 +89,18 @@ impl Index {
       .map(|(height, _hash)| height + 1)
       .unwrap_or(0);
 
-    let outputs_indexed = wtx.open_table(OUTPOINT_TO_ORDINAL_RANGES)?.len()?;
+    let utxos_indexed = wtx.open_table(OUTPOINT_TO_ORDINAL_RANGES)?.len()?;
+
+    let outputs_traversed = wtx
+      .open_table(OUTPUTS_TRAVERSED)?
+      .get("outputs_traversed")?
+      .unwrap_or(0);
```

Comment:
I think this `.into()` will work, not sure:

```suggestion
    let outputs_traversed = wtx
      .open_table(STATISTICS)?
      .get(Statistics::OutputsTraversed.into())?
      .unwrap_or(0);
```